client/src/pages/RoomPage.tsx

```diff
@@ -57,14 +63,18 @@ function RoomPage({ name }: { name: string }): JSX.Element {
     removeAvatar(id);
   };
 
+  const printPack = (pack) => {
+    console.debug(pack);
+  };
+
   // open all listeners on render
   useEffect(() => {
     notifyAndRequestNetworkInfo(socket, name);
-    openJoinListener(socket, onJoin);
+    openJoinListener(socket, onNewJoin);
     openLeaveListener(socket, setAnnouncement, onLeave);
     openRequestUsersListener(socket, setNewUser);
-    openOfferListener(getUsers(), socket);
-    openAnswerListener(getUsers(), socket);
+    openOfferListener(getUsers(), socket, printPack, printPack);
```

Comment:
(seems dumb

Reply:
True, it was just for debugging purposes so I'll get rid of it.